fdbserver/include/fdbserver/DDRelocationQueue.h

```diff
@@ -122,6 +119,8 @@ struct DDQueueInitParams {
 
 // DDQueue receives RelocateShard from any other DD components and schedules the actual movements
 class DDQueue : public IDDRelocationQueue, public ReferenceCounted<DDQueue> {
+	const DDEnabledState* ddState = nullptr;
```

Comment:
Nit: Would it make sense to also call this `ddEnabledState` so that a reader can quickly understand what this refers to while reading the code in `DDRelocationQueue.actor.cpp`?